src/olympia/blocklist/tasks.py

```diff
@@ -78,8 +78,20 @@ def import_block_from_blocklist(record):
         log.debug(
             'Kinto %s: Attempting to create Blocks for addons matching [%s]',
             kinto_id, guid_regexp)
-        addons_guids_qs = Addon.unfiltered.using(using_db).filter(
-            guid__regex=guid_regexp).values_list('guid', flat=True)
+        # we're going to try to split the regex into a list for efficiency.
+        guids_list = split_regex_to_list(guid_regexp)
+        if guids_list:
+            log.debug(
+                'Kinto %s: Broke down regex into list of guids [%s]',
+                kinto_id, guids_list)
+            addons_guids_qs = Addon.unfiltered.using(using_db).filter(
+                guid__in=guids_list).values_list('guid', flat=True)
+        else:
+            log.debug(
+                'Kinto %s: Unable to break down regex into list of guids',
+                kinto_id)
```

Comment:
I found it helped me to debug what was going on better to add the `guid` in that log, and remove it from the `Attempting to create Blocks` log above. Makes it consistent with the case  where we did manage to break down the regexp into a list, too (where we log the final list)

Reply:
https://github.com/mozilla/addons-server/pull/13810/commits/4a915dd0ec8991fe488100d574a6e9b42d4b76e0